packages/zephyr/src/Button.tsx

```diff
@@ -10,7 +10,7 @@ import styled, { keyframes, useTheme } from 'styled-components';
 import { Box, BoxStylingProps } from './Box';
 import { ButtonVariantName } from './theme/variants/button';
 
-export type ButtonSize = 'large' | 'medium' | 'small';
+export type ButtonSize = 'xs' | 'small' | 'medium' | 'large';
```

Comment:
```suggestion
export type ButtonSize = 'extra-small' | 'small' | 'medium' | 'large';
```

rather have `x-small` or `extra-small` or have everything changed to `xs` `s` `m` `l`

Reply:
Will make the change!